Utility/Remove-ResourceGroups.ps1

```diff
@@ -0,0 +1,228 @@
+﻿
+<#PSScriptInfo
+
+.VERSION 1.0
+
+.GUID 
+
+.AUTHOR Azure-Automation-Team
+
+.COMPANYNAME Microsoft
+
+.COPYRIGHT 
+
+.TAGS AzureAutomation Utility
+
+.LICENSEURI 
+
+.PROJECTURI https://github.com/azureautomation/runbooks/blob/master/Utility/Remove-ResourceGroups.ps1
+
+.ICONURI 
+
+.EXTERNALMODULEDEPENDENCIES 
+
+.REQUIREDSCRIPTS 
+
+.EXTERNALSCRIPTDEPENDENCIES 
+
+.RELEASENOTES
+
+#>
+<# 
+.SYNOPSIS 
+  Connects to Azure and removes all resource groups which match the name filter
```

Comment:
You may want to mention that this deletes the resource groups and the child resources so users should be very careful that they don't accidently delete any resources.